crates/ty_python_semantic/src/types/infer.rs

```diff
@@ -2315,6 +2316,13 @@ impl<'db, 'ast> TypeInferenceBuilder<'db, 'ast> {
                         continue;
                     }
                 }
+                Type::KnownInstance(KnownInstanceType::Deprecated(deprecated_inst)) => {
+                    // This isn't picked up by FunctionDecorators so we need to continue here
+                    // to avoid having the KnownInstanceType get type checked as it's not callable
+                    // FIXME: we shouldn't be losing the fact that a KnownInstance *is* callable!
```

Comment:
You can fix this by adding a new
```rs
Type::KnownInstance(known_instance @ KnownInstanceType::Deprecated(_)) => {
    known_instance.instance_fallback(db).bindings(db)
}
```
branch to `Type::bindings`.

Reply:
Oh perfect!